packages/node_modules/@webex/plugin-meetings/src/media/detectSilence/audiocontext.js

```diff
@@ -0,0 +1,88 @@
+/* globals webkitAudioContext, AudioContext */
+/* eslint-disable no-underscore-dangle */
+
+// eslint-disable-next-line no-nested-ternary
+const NativeAudioContext = typeof AudioContext !== 'undefined' ?
```

Comment:
I'm not sure if it's worth trying to fallback to webkitAudioContext. According to https://developer.mozilla.org/en-US/docs/Web/API/Web_Audio_API/Migrating_from_webkitAudioContext there are many differences between webkitAudioContext and AudioContext and I think all the browsers we care about (Chrome, FF, Safari) support AudioContext already, don't they?